imap/sieve_db.c

```diff
@@ -935,6 +937,7 @@ EXPORTED int sieve_ensure_folder(const char *userid, struct mailbox **mailboxptr
                    mboxname, error_message(r));
             goto done;
         }
+	(*mailboxptr)->silentchanges = 1;
```

Comment:
Is this a typo? I would expect it to be `... = silent;` rather than `... = 1;`, but maybe I've missed a detail earlier

Reply:
I have the same question/concern

Reply:
This here is responsible for the CRC madness!  Yes, you're right.